src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,144 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    static class Node<T> {
+        private T element;
+        private Node<T> prev;
+        private Node<T> next;
+
+        public Node(T element) {
+            this.element = element;
```

Comment:
Write correct constructor. Your Node includes references to previous and next Nodes and value

Reply:
in this case, it doesn't matter